TCAT/Utilities/SearchTableViewHelpers.swift

```diff
@@ -24,6 +24,7 @@ enum SectionType {
     case seeAllStops
     case searchResults
     case currentLocation
+    case favorites
 }
 
```

Comment:
small thing but can u get rid of line 18?  feels there shouldn't be a space there